recipes/gridfill/bld.bat

```diff
@@ -0,0 +1,10 @@
+set CFG=%USERPROFILE%\pydistutils.cfg
+echo [config] > "%CFG%"
+echo compiler=mingw32 >> "%CFG%"
+echo [build] >> "%CFG%"
+echo compiler=mingw32 >> "%CFG%"
+echo [build_ext] >> "%CFG%"
+echo compiler=mingw32 >> "%CFG%"
```

Comment:
This weird workaround got the Fortran extension compiling with `mingwpy` on `Win32` and  `Python <3.5`.